### ISSUE TITLE
Titlepage issue with "type" option in English vs. German

### ISSUE THREAD

**Issue by MillieIS** [tudace/tuda_latex_templates#194]
I have my main babel language set to english with ngerman as an additional package. When using main=english, regardless of what I set the type to in the \documentclass (i.e. type=dr or type=drfinal) the output on this 2nd title page is just "doctoral thesis" rather than "eingreichte" or "vorgelegte" Dissertation. If I change my main babel language to ngerman, however, the title page has the proper text (i.e. with the option type=drfinal in \documentclass{tudapub} and the option main=english in \usepackage{babel} it is _incorrect,_ with the option type=drfinal in \documentclass{tudapub} and the option main=ngerman in \usepackage{babel} it is _correct_, as shown in the images, respectively.) 

<img width="1004" alt="1-babel_main_english" src="https://user-images.githubusercontent.com/25899279/83863522-bc1e0780-a723-11ea-97fa-811a9912577a.png">
<img width="1004" alt="2-babel_main_ngerman" src="https://user-images.githubusercontent.com/25899279/83863532-be806180-a723-11ea-9b8e-5899d2ed9a68.png">

My implementation is a simplified preamble is the following
```
\documentclass[
	english,
	ruledheaders=chapter,
	class=book,
	thesis={
		type=drfinal,
		dr=ing,
		department=mb
	},
	instbox=false,
	accentcolor=1a,
	custommargins=true,
	marginpar=false,
	fontsize=11pt,
]{tudapub}

\usepackage[main=english, ngerman]{babel}
```

**Comment by schoeps** [comment]
I agree that this seems to be a bug because German and English template should behave similarly. 

However, I am not sure if the distinction between "dr" and "drfinal" is correct in German. 

@marklaz do you recall where this comes from? It's not explicitily mentioned in https://www.tu-darmstadt.de/media/dezernat_ii/promotionen_dokumente/Dissertation-Titelblatt.de.pdf or https://www.intern.tu-darmstadt.de/media/dezernat_ii/promotionen_dokumente/Dissertation-Titelblatt.en.pdf. On the other hand, Machinenbau (https://www.maschinenbau.tu-darmstadt.de/media/maschinenbau/dokumente_2/doktoranten/promotion/promotionsleitfaden.pdf, page 26) uses the German terms "eingereichte" and "genehmigte" instead of "eingreichte" and "vorgelegte" which makes more sense to me?

**Comment by marklaz** [comment]
Page 11 of the Maschinenbau document states the difference between the two stages of the dissertation:
- “eingereicht” when submitting at the department
- “genehmigt” when submitting at ULB after defense.
But probably that is not the main point here.

In “Dissertation-Titelblatt.de.pdf” it seems to me, as if a verb is missing.

In English, the difference of the versions might be something like “submitted“ and “accepted”.
—————————————————————————-
Nevertheless, I agree that there is some bug in the template.
Summing up, we require: 
- type=dr -> “eingereicht” (“submitted”)
- type=drfinal -> “genehmigt” (“accepted”)

@TeXhackse:
Is that fine for you, or do you require any additional information for updating the template?

**Comment by TeXhackse** [comment]
@marklaz we had talked about this. Since we enforced the titlepage to be German we did not translate it- Which was a decision long long time ago. Was together with my question, because I was not sure, if we have to mention it. But ofc we can add it.

**Comment by schoeps** [comment]
There are two issues: 

1. German: `drfinal` should give "genehmigte" instead of "vorgelegte". I hope everyone agrees.

2. English: according to the poster both options (`dr` and `drfinal`) give the same translation "doctoral thesis" (on page 2?), so there is already a translation happening and this should reflect the setting `dr` vs `drfinal`.

**Comment by TeXhackse** [comment]
alright. will fix it.

**Comment by TeXhackse** [comment]
@marklaz @schoeps : Sorry, did not find time earlier, but I got to add, that this
> Nevertheless, I agree that there is some bug in the template.
Summing up, we require:

    type=dr -> “eingereicht” (“submitted”)
    type=drfinal -> “genehmigt” (“accepted”)

already is correct. At least for my setup, if anyone of you, and this includes @MillieIS did not get this that way, I'd ask for an example, including logfiles. There's something going wrong then. I cannot reconstruct it. 

Anyway I now added the english language and will prepare a release, but wait for an answer concerning this, to fix it at once.

**Comment by marklaz** [comment]
To my understanding, the problem occurred only in the English version, as there has been no additonal participle (like "submitted" or "accepted") before "doctoral thesis".

**Comment by TeXhackse** [comment]
@schoeps  said: 

> There are two issues:
> 
>     1. German: `drfinal` should give "genehmigte" instead of "vorgelegte". I hope everyone agrees.

and this is something I cannot reconstruct.

**Comment by marklaz** [comment]
If "drfinal" gives "genehmigte" in German already, everything should be fine.

**Comment by TeXhackse** [comment]
okay then, we had a missunderstanding. Just wanted to ensure this. rechecking the other issues and then i will do an upload

**Comment by schoeps** [comment]
> @schoeps said:
> 
> > There are two issues:
> > ```
> > 1. German: `drfinal` should give "genehmigte" instead of "vorgelegte". I hope everyone agrees.
> > ```
> 
> and this is something I cannot reconstruct.

Sorry, this was claimed by the original poster. I did not check it myself.

**Comment by TeXhackse** [comment]
Okay then. Release incoming.

**Comment by MillieIS** [comment]
> @schoeps said:
> 
> > There are two issues:
> > ```
> > 1. German: `drfinal` should give "genehmigte" instead of "vorgelegte". I hope everyone agrees.
> > ```
> 
> and this is something I cannot reconstruct.

Yes, sorry, I mispoke, but in the example I gave in the original post it gives "genehmigte" as long as main language is set to german.

**Comment by TeXhackse** [comment]
It's no problem. I just wanted to ensure, that I don't miss anything about it.

Release is published and uploaded to CTAN. Will be available soon.
